db/seeds.rb

```diff
@@ -5,3 +5,75 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+require "faker"
+
+users_created = 0
+instrument_categories_created = 0
+instruments_created = 0
+reviews_created = 0
+bookings_created = 0
+
+for i in (1..5)
+  user = User.new do |u|
+    users_created += 1
+    u.email = Faker::Internet.email
+    u.password = Faker::Internet.password
+    u.created_at = Faker::Date.backward(days: 14)
+    u.updated_at = Faker::Date.backward(days: 14)
+
+    for i in (1..2)
+      InstrumentCategory.new do |instcat|
+        instrument_categories_created += 1
+        instcat.category = Faker::Music.instrument
+        instcat.created_at = Faker::Date.backward(days: 14)
+        instcat.updated_at = Faker::Date.backward(days: 14)
+
+        for i in (1..2)
+          instrument = Instrument.new do |inst|
+            instruments_created += 1
```

Comment:
we also need some names, description, price and type for the instruments

**my suggestions**

Name:
Faker::Lorem.sentence(word_count: 3) #=> "Quaerat quam unde."

Description:
Faker::Lorem.paragraph #=> "Recusandae minima consequatur. Expedita sequi blanditiis. Ut fuga et."

Price:
Faker::Number.between(from: 10, to: 50)

type:
Faker::Music.instrument #=> "Ukelele"

Reply:
Added all of those :)